src/api/mod.rs

```diff
@@ -1226,6 +1226,58 @@ impl<'a> AuthenticatedApi<'a> {
         Ok(rv)
     }
 
+    /// Fetch organization events from the specified dataset
+    pub fn fetch_organization_events(
```

Comment:
Linter is already complaining about this, but I also noticed this seems like too many function arguments, and I would therefore suggest combining them into a struct